### ISSUE TITLE
Here's how you know accordion does not have meaningful accessible name describing it's purpose.

### ISSUE THREAD

**Issue by JohnHewittCriterion** [flexion/fs-intake-module#832]
The here's how you know accordion does not tell the screen reader user what exactly they will "know about" after opening the control so the purpose is not really described meaningfully.
 
Modify the accessible name with aria-label or aria-labelledby http://pauljadam.com/demos/positionbug.html
 e.g. aria-label="Here's how you know we're an official website of the US government." 

![image](https://user-images.githubusercontent.com/28604890/30262574-70dd5876-969f-11e7-8f10-54ecc87f83b7.png)

WCAG: 2.4 Navigable - 2.4.6 Headings and Labels 
EPERM-157

Tasks
- [ ] Discuss possibility of hiding the side nav from screen readers (John, Seth & Chris)

**Comment by npage123** [comment]
- [ ] Check with Mac Voice Over
- [ ] Check with JAWS
